x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/register_alerts_table_configuration.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { type TriggersAndActionsUIPublicPluginSetup } from '@kbn/triggers-actions-ui-plugin/public';
+import type {
+  AlertsTableConfigurationRegistry,
+  RenderCustomActionsRowArgs,
+} from '@kbn/triggers-actions-ui-plugin/public/types';
+import { EuiDataGridColumn } from '@elastic/eui';
+import { SortCombinations } from '@elastic/elasticsearch/lib/api/types';
+import type { SortOrder } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import {
+  ALERT_DURATION,
+  ALERT_END,
+  ALERT_REASON,
+  ALERT_RULE_NAME,
+  ALERT_START,
+  ALERT_STATUS,
+} from '@kbn/rule-data-utils';
+import type { FieldFormatsRegistry } from '@kbn/field-formats-plugin/common';
+import { getAlertFlyout } from './use_alerts_flyout';
+import {
+  ALERT_ANOMALY_DETECTION_JOB_ID,
+  ALERT_ANOMALY_SCORE,
+  ALERT_ANOMALY_TIMESTAMP,
+} from '../../../common/constants/alerts';
+import { getAlertFormatters, getRenderCellValue } from './render_cell_value';
+import { AlertActions } from './alert_actions';
+
+export function registerAlertsTableConfiguration(
+  triggersActionsUi: TriggersAndActionsUIPublicPluginSetup,
+  fieldFormats: FieldFormatsRegistry
+) {
+  const columns: EuiDataGridColumn[] = [
+    {
+      id: ALERT_STATUS,
+      displayAsText: i18n.translate('xpack.ml.alertsTable.columns.status', {
+        defaultMessage: 'Status',
+      }),
+      initialWidth: 150,
+    },
+    {
+      id: ALERT_REASON,
+      displayAsText: i18n.translate('xpack.ml.alertsTable.columns.reason', {
+        defaultMessage: 'Reason',
+      }),
+      initialWidth: 150,
+    },
+    {
+      id: ALERT_RULE_NAME,
+      displayAsText: i18n.translate('xpack.ml.alertsTable.columns.ruleName', {
+        defaultMessage: 'Rule name',
+      }),
+      initialWidth: 150,
+    },
+    {
+      id: ALERT_ANOMALY_DETECTION_JOB_ID,
+      displayAsText: i18n.translate('xpack.ml.alertsTable.columns.jobId', {
+        defaultMessage: 'Job ID',
+      }),
+      initialWidth: 150,
+    },
+    {
+      id: ALERT_ANOMALY_SCORE,
+      displayAsText: i18n.translate('xpack.ml.alertsTable.columns.anomalyScore', {
+        defaultMessage: 'Latest anomaly score',
+      }),
+      initialWidth: 150,
+      isSortable: true,
+      schema: 'number',
```

Comment:
Do you know why this is showing up as a `text` field in the sorting controls?

<img width="589" alt="image" src="https://github.com/elastic/kibana/assets/7405507/c8fc2954-b1f8-4966-bdf1-a45fd9751172">

Reply:
Most likely because it is not defined in the mapping

Reply:
Fixed in af1b583c7dbf439317ca657ee79899529fd1e68d